scripts/Muon/GUI/Common/context_example/context_example_view.py

```diff
@@ -8,22 +8,21 @@
 from __future__ import (absolute_import, division, print_function)
 
 
-from PyQt4 import QtCore
-from PyQt4 import QtGui
+from qtpy import QtCore, QtWidgets
```

Comment:
were not using this file, so there was no need to change it.

Reply:
Either way, using or not, I don't think we should have code that is out of date for our current purposes. I believe the policy is if it's not used, delete it, should I just delete it?